dali/ft/filecopy.cpp

```diff
@@ -592,6 +596,24 @@ FileSprayer::FileSprayer(IPropertyTree * _options, IPropertyTree * _progress, IR
     encryptKey.set(options->queryProp(ANencryptKey));
     decryptKey.set(options->queryProp(ANdecryptKey));
 
+    fileUmask = -1;
+    if (options->hasProp(ANumask))
+    {
+        StringBuffer umaskStr;
+        options->getProp(ANumask, umaskStr);
```

Comment:
Yes, good point, would make it cleaner (not sure that handles invalid characters any better tho?)